src/cms/views/feedback/feedbackList_view.py

```diff
@@ -0,0 +1,129 @@
+from django.shortcuts import render
+from django.views.generic import TemplateView
+from cms.forms.feedback.feedback import FeedbackForm
+from cms.models.feedback.event_feedback import EventFeedback
+from cms.models.feedback.offer_feedback import OfferFeedback
+from cms.models.feedback.region_feedback import RegionFeedback
+from cms.models.feedback.page_feedback import PageFeedback
+
+
+class FeedbackView(TemplateView):
```

Comment:
```suggestion
class FeedbackListView(TemplateView):
```
Please keep the class names and file names consistent with the others in this project.
In this case, the class name should be `FeedbackListView`, and the filename should be `feedback_list_view.py`.